<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>resultDataSourceDialog</class>
 <widget class="QDialog" name="resultDataSourceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>411</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Result Data Source</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Data Source:</string>
    </property>
    <property name="buddy">
     <cstring>dsCombo</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="dsCombo">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Data Mapping</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Scale:</string>
    </property>
    <property name="buddy">
     <cstring>dsScale</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Bias:</string>
    </property>
    <property name="buddy">
     <cstring>dsBias</cstring>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dsScale">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-100000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000.000000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dsBias">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-100000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="cbVis">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>371</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Visible on the plot</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>resultDataSourceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>resultDataSourceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
